src/config.rs

```diff
@@ -65,6 +65,14 @@ pub enum Config {
     #[strum(props(default = "0"))]
     KeyGenType,
 
+    /// Timer in seconds after which the message is deleted from the
+    /// server.
+    ///
+    /// Equals to -1 by default, which means the message is never
```

Comment:
i do not really like the -1 here. i would suggest 0=off, 1=atOnce, >1 seconds

Reply:
Fixed it by adding one more commit.